consensus/src/subscriber.rs

```diff
@@ -28,11 +25,11 @@ pub struct SubscriberHandler {
     /// Receive reconfiguration update.
     rx_reconfigure: watch::Receiver<ReconfigureNotification>,
     // Channel to receive the output of consensus.
-    rx_sequence: Receiver<ConsensusOutput>,
+    rx_sequence: metered_channel::Receiver<ConsensusOutput>,
```

Comment:
Maybe you want to make the `metered_channel` `Sender` / `Receiver` the default you use in this scope, unless you also use a lot of `mpsc`?

Reply:
Good point, will do that!